api/mmlogic.proto

```diff
@@ -17,22 +17,30 @@ package api;
 option go_package = "internal/future/pb";
 
 import 'api/messages.proto';
+import 'google/api/annotations.proto';
 
-message RetrievePoolRequest{
+message RetrievePoolRequest {
   // The Pool representing the set of Filters to be queried.
-  messages.Pool pool = 1;
+  Pool pool = 1;
 }
 
-message RetrievePoolResponse{
+message RetrievePoolResponse {
   // The Tickets that meet the Filter criteria requested by the Pool.
-  repeated messages.Ticket ticket = 1;
+  repeated Ticket ticket = 1;
 }
 
-// The MMLogic API provides utility functions for common MMF functionality such as retreiving Tickets
-// from state storage.
+// The MMLogic API provides utility functions for common MMF functionality such
+// as retreiving Tickets from state storage.
 service MmLogic {
-  // RetrievePool gets the list of Tickets that match every Filter in the specified Pool.
+  // RetrievePool gets the list of Tickets that match every Filter in the
+  // specified Pool.
+  // TODO: Consider renaming to "GetPool" to be consistent with HTTP REST CRUD
+  // conventions.
   rpc RetrievePool(RetrievePoolRequest) returns (stream RetrievePoolResponse) {
+    option (google.api.http) = {
+      post: "/v1/mmlogic/pools"
```

Comment:
I think it is a bit confusing to set up a POST endpoint for a Get operation, is it possible to use a GET endpoint instead?